modules/auxiliary/scanner/http/splunk_web_login.rb

```diff
@@ -184,4 +183,29 @@ def do_login(user, pass)
     end
   end
 
+  def report_cred(opts)
+    service_data = {
+      address: opts[:ip],
+      port: opts[:port],
+      service_name: opts[:service_name],
+      protocol: 'tcp',
+      workspace_id: myworkspace_id
+    }
+
+    credential_data = {
+      origin_type: :service,
+      module_fullname: fullname,
+      username: opts[:user],
+      private_data: opts[:password],
+      private_type: :password
+    }.merge(service_data)
+
+    login_data = {
+      core: create_credential(credential_data),
+      status: Metasploit::Model::Login::Status::UNTRIED,
```

Comment:
If you have a successful login shouldn't this be `Metasploit::Model::Login::Status::SUCCESSFUL`?